cpp/src/arrow/util/compression_snappy.cc

```diff
@@ -95,6 +92,4 @@ class SnappyCodec : public Codec {
 
 std::unique_ptr<Codec> MakeSnappyCodec() { return std::make_unique<SnappyCodec>(); }
 
-}  // namespace internal
-}  // namespace util
-}  // namespace arrow
+} // namespace arrow::util::internal
```

Comment:
I suspect this lack of newline will cause a clang-format error

Reply:
Hmmm , Let me fix this as well